ceph/UI/helper.py

```diff
@@ -0,0 +1,23 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+def take_ss(driver, filename: str):
+    """Takes a screenshot and saves it to the specified filename."""
+    try:
+        driver.save_screenshot(filename)
+    except Exception as e:
+        raise RuntimeError(f"Failed to take screenshot: {e}")
+
+
+def wait_for_element(driver, element_id: str, timeout: int = 10):
```

Comment:
Extend the logic for all element types like class, name, xpath, css selector etc and not just to id.